WordPress/gutenberg
# Font Library: create font faces through the REST API

## What?

Continuing the work started in https://github.com/WordPress/gutenberg/pull/57656 to build out the Font Faces REST API controller

- Returning proper response data
- Creating font faces, including uploading files

## Testing Instructions

Test the following routes and ensure the proper responses

```
GET wp/v2/font-families/<id>/font-faces
GET wp/v2/font-families/<id>/font-faces/<id>
POST wp/v2/font-families/<id>/font-faces
DELETE wp/v2/font-families/<id>/font-faces/<id>
```

The POST request requires using multipart/form-data to include a font file. Here's an example of using the Postman client to make a request

<img width="1084" alt="image" src="https://github.com/WordPress/gutenberg/assets/1699996/2d60eb1a-9dd6-4410-bd2a-dcb0364afb97">

(you can "Copy as curl" from your browser network tab to get the cookies, and then "Import" that into Postman to make an authenticated request)
